app/assets/javascripts/common.js

```diff
@@ -55,4 +61,9 @@ $(document).ready(function () {
             });
         }
     });
+
+    // If an Info/Assess block was just added, open that item in the accordion
+    if (window.location.search.match('edit_')) {
+      $('.accordion_embeddables').accordion('option', 'active', -1);
+    }

```

Comment:
@scytacki I'm especially wondering what you think about this check that I added. It's how I set it up to open the most recently added info/assessment item in the accordion. It checks for the 'edit_[type]' parameter that was already being set whenever an item is added. I'm wondering though if I should add a new parameter that could be used to specify that a particular item in the accordion should be open on page load. Would that be better?

Reply:
I think this depends on what we do about automatically opening the edit dialog, that I commented on below.

If you do, do this check, then it seems you should be checking the actual parameter not just the full url. And you should be checking the id value of the parameter, like the code for automatically opening the dialog is trying to do.

Also another case to consider is what happens after someone edits an item. The page will be reloaded in this case. It seems natural to keep the item still expanded after doing this. However the item might not be the last one on the page.

Reply:
@emcelroy somehow your comment below ended up in the wrong thread. I'm copying it here
> It doesn't matter that much, but that code window.location.search.match('edit_') isn't looking for a match in the full URL. It's looking for a match in window.location.search which is only the query string part of the URL. Since the edit parameter name varies depending on which assessment type you add, I thought that was the simplest way to just check that an item of any type had been added. I do understand it's not ideal, but the edit param names vary according to which type of assessment item you add and I was wary of adding something to check all possible edit param name possibilities.

> Anyway, I think the code you pointed to in _embedded_editor_links.html.haml in a previous comment is a better example to follow. I'll do something like that instead.

> Also FWIW: When you edit an existing item, an edit param does not get added to the URL when the page reloads. It's only when you add a new item. Maybe the param should be added when you edit something, too?

Reply:
Sorry I missed the 'search' part of that code above. 

The point of edit param (which Doug is fixing) is that it is supposed to trigger the edit popup to show automatically on page load.  So this would be bad if we do it after closing the editing dialog since that would just open it again.

I think at this point it better not to worry about re-opening the last edited embeddable. It is more important to get this in a place where people can demo it.